reana_commons/snakemake.py

```diff
@@ -157,7 +159,11 @@ def files(items):
             omitrules=omitrules,
         )
 
-        workflow.persistence = Persistence(dag=dag)
+        if hasattr(workflow, "_persistence"):
+            workflow._persistence = Persistence(dag=dag)
+        else:
+            # for backwards compatibility (Snakemake < 7 for Python 3.6)
+            workflow.persistence = Persistence(dag=dag)
```

Comment:
This is needed to also support Python 3.6, that uses a Snakemake version (6.15) in which the new `default_target` directive is present, but internally `persistence` had not been replaced by `_persistence` yet in the `snakemake.Workflow` class. I think this approach of directly checking the attribute (rather than checking, say, the Python version) is correct, but I'm more than happy to get advice on how to do this better, if you think there's a more appropriate way!